src/tools/pub/pubspec.md

```diff
@@ -398,6 +403,43 @@ the pattern is considered relative to the package's root directory.
   Support for the `false_secrets` field was added in Dart 2.15.
 {{site.alert.end}}
 
+### Screenshots
+
+Packages may showcase their widgets or other visual elements using screenshots.
+
+A package can include up to 10 screenshots. 
+These are specified in the pubspec as a list under the `screenshots` field. 
+Each screenshot is specified by a `description` and a `path`. 
+The `description` is a string of at most 160 characters
+describing the screenshot. 
+For example:
+
+```yaml
+screenshots:
+  - description: 'description of screenshot'
+    path: path/to/image/in/package/500x500.webp
+  - description: 'description of second screenshot'
+    path: path/to/image/in/package.png
+```
+
+Supported file types include: `png`, `jpg`, `gif`, and `webp`. 
+Both static and animated files are supported. 
+The screenshots shown on the pub.dev site
+will all be `webp` images generated from the original image. 
+
+The first screenshot in the list will be used for a thumbnail. 
+Thumbnails will always be static. 
+In case the original image is animated,
+the first frame will be used for the thumbnail. 
+Pub.dev generates square thumbnails, so, 
+for the first screenshot, 
+we recommend a 1:1 aspect ratio. 
+If the ratio is not 1:1, 
+the image will be resized to fit in a square, 
+but keeping the original aspect ratio. 
```

Comment:
What does this mean? Does it add margins to the image?

Reply:
It doesn't add margins. The image is scaled down such that the largest dimension is reduced to the size of the square. And then the other dimension is resized according to the aspect ratio. (The image will appear in the center of the square.) Does it make sense? :-)

Reply:
In any case I removed this section. See my comment below.